src/Symfony/Component/Lock/Tests/Serializer/LockKeyNormalizerTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Symfony\Component\Lock\Tests\Serializer;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Lock\Exception\UnserializableKeyException;
+use Symfony\Component\Lock\Key;
+use Symfony\Component\Lock\Serializer\LockKeyNormalizer;
+
+class LockKeyNormalizerTest extends TestCase
+{
+    private LockKeyNormalizer $normalizer;
+
+    protected function setUp(): void
+    {
+        $this->normalizer = new LockKeyNormalizer();
+    }
+
+    public function testNormalize()
+    {
+        $key = new Key(__METHOD__);
+        $key->reduceLifetime(1);
+        $key->setState('foo', 'bar');
+
+        $copy = $this->normalizer->denormalize($this->normalizer->normalize($key), Key::class);
+        $this->assertSame($key->getState('foo'), $copy->getState('foo'));
+        $this->assertEqualsWithDelta($key->getRemainingLifetime(), $copy->getRemainingLifetime(), 0.001);
+    }
+
+    public function testDenormalize()
```

Comment:
This test name as bad as this test is not covering denormalization at all.

Reply:
Thanks, fixed